packages/environment-check/src/output/zip.ts

```diff
@@ -20,6 +21,32 @@ function byteNumberToSizeString(byteNumber: number): string {
     return `${parseFloat((byteNumber / Math.pow(1024, i)).toFixed(2))} ${units[i]}`;
 }
 
+/**
+ * Convert results to JSON.
+ *
+ * @param results - environment check results
+ * @returns json string
+ */
+function convertResultsToJson(results: EnvironmentCheckResult): string {
```

Comment:
not sure whether we need to massage the result json. I think I would keep it raw, it contains the information of `requestedChecks` so when checking the JSON it is should be clear what checks have been executed

Reply:
Ok. I've reverted